ci/matrix.xml

```diff
@@ -277,6 +278,30 @@
         'set scalaJSUseRhino in Global := true' test
   ]]></task>
 
+  <task id="sbtplugin-test-sbt1"><![CDATA[
```

Comment:
Any reason you are not generalizing the `sbtplugin-test` task? The CI script supports any variable. Just use `$sbt` and set it with a `<v>` tag.